djitellopy/tello.py

```diff
@@ -772,31 +788,32 @@ def send_rc_control(self, left_right_velocity: int, forward_backward_velocity: i
             up_down_velocity: -100~100 (up/down)
             yaw_velocity: -100~100 (yaw)
         """
-        def round_to_100(x: int):
-            if x > 100:
-                return 100
-            if x < -100:
-                return -100
-            return x
+        def round_to_max(x: int):
+            MAX = 100
+            return -MAX if x < -MAX else min(x, MAX)
 
         if time.time() - self.last_rc_control_timestamp > self.TIME_BTW_RC_CONTROL_COMMANDS:
             self.last_rc_control_timestamp = time.time()
-            self.send_command_without_return('rc %s %s %s %s' % (round_to_100(left_right_velocity),
-                                                                round_to_100(forward_backward_velocity),
-                                                                round_to_100(up_down_velocity),
-                                                                round_to_100(yaw_velocity)))
+            SCHEMA = 'rc %s %s %s %s'
+            self.send_command_without_return(SCHEMA % (round_to_100(left_right_velocity),
```

Comment:
PEP-8 claims all rount_to_100 shall be aligned in the same column. Here all except the first one are aligned.

Reply:
There's a one liner terminal command to format it, can you recall it? Otherwise I'll search on StackExchange, I know it's there. Note: If your editor didn't use it, this will also do formatting changes in other places of the file.